.serena/memories/notifications.md

```
@@ -0,0 +1,41 @@
+## Delivery Model (Fire-and-Forget)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)